custom_components/givenergy_local/dashboard.py

```
@@ -78,20 +78,35 @@ def warn_line(y: float, text: str) -> dict:
-                       tailored view set (EMS scheduling controls + integration health).
+        inv:                   Inverter (or EMS controller) serial number (lowercase).
+        bats:                  Battery serial number(s) (lowercase).
+        max_power_kw:          Vertical envelope (±kW) applied to the Overview power chart's
+                               y-axis. Defaults to 10 kW — suitable for single-phase hybrid
+                               and AC-coupled inverters. Raise for larger 3-phase systems.
+        is_ems:                True when generating for an EMS plant controller. An EMS has no
+                               PV/battery/grid/load sensors or inverter controls, so the
+                               inverter-centric views would render blank — instead emit a
+                               tailored view set (EMS scheduling controls + integration health).
+        has_ac_config_block:   True for AC-coupled / All-in-One plants that expose the
+                               HR(300–359) AC-output config block (export priority, EPS,
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)